### ISSUE TITLE
Replay Subject Issue

### ISSUE THREAD

**Issue by ashfurrow** [ashfurrow/Action#3]
Waiting on https://github.com/ReactiveX/RxSwift/pull/276

**Comment by ashfurrow** [comment]
So the issue is actually that we have a replay subject of 10, which is "big enough", but soon it'll be unlimited. 

Why bother? Well here's the setup:

- `workFactory` produces an observable that sends > 10 items immediately. 
- You subscribe to next events on the observable that `execute()` returns, but it's already been pumping out events before your subscription.
- You only get the 10 most recent next events when you do subscribe. 

It's a chicken-and-egg problem, should be fixed soon :hatching_chick:

**Comment by fpillet** [comment]
`replayAll()` support has now been merged into RxSwift - just need to wait for the next beta

**Comment by ashfurrow** [comment]
Thanks for the update!
